itest/litd_custom_channels_test.go

```diff
@@ -4512,7 +4996,7 @@ func testCustomChannelsDecodeAssetInvoice(ctx context.Context,
 	}
 
 	const decimalDisplay = 6
-	itestAsset = &mintrpc.MintAsset{
+	tcAsset := &mintrpc.MintAsset{
```

Comment:
That's a great catch from @jtobin. Shouldn't we also fix this for `testCustomChannelsOraclePricing`? Because that's also a test that mutates the global `itestAsset` variable. I would also add a godoc comment to `itestAsset` to explain that we should consider it immutable.

```
// itestAsset is a predefined asset that is used in the tests below. It
// should be considered immutable.
```